src/App.css

```diff
@@ -48,19 +46,18 @@ ul {
 }
 
 .link {
-  background-color: #e1dada;
-  border-radius: 5px;
+  background: transparent;
+  border: 1px solid black;
+  border-radius: 10px;
   color: black;
-  text-align: center;
-  text-decoration: none;
-  padding: 5px;
-  min-width: 84px;
-  border: none;
   font-size: 16px;
+  padding: 7px;
+  text-decoration: none;
 }
 
 .welcome-button {
   width: 100%;
+  margin-bottom: 10px;
```

Comment:
Only a small opinion: 

I find [the "relative" units](https://thecssworkshop.com/lessons/relative-units) are more flexible (`%`, `em`, `rem`, etc).   I really like `rem` units. Overall if you are thinking of multiple screen sizes. **Don't require changes.** They are only ideas 😄 !

Reply:
Thanks for the advice, I think because of the order of the project, we are all using px and % when needed, so for this time it might be better to have it that way, otherwise, I love your recommendation because I have never used this relative unit, so I will investigate and use it in my future projects 👏

Reply:
No problem, Susana! 👍🎉💯

I'm so happy you consider to research more it. 🎉💯 This an interesting topic.